test/mocks/api_server.go

```diff
@@ -367,6 +369,10 @@ func (b *MockConfigurableServerBuilder) SetGetOrganizationQuotaCost(list *amsv1.
 	b.handlerRegister[EndpointOrganizationQuotaCostGet] = buildMockRequestHandler(list, err)
 }
 
+func (b *MockConfigurableServerBuilder) SetGetOrganizations(list *amsv1.OrganizationList, err *ocmErrors.ServiceError) {
```

Comment:
```suggestion
func (b *MockConfigurableServerBuilder) SetOrganizationsGetResponse(list *amsv1.OrganizationList, err *ocmErrors.ServiceError) {
```

Reply:
can we also add a comment with the url for which the response is set?

Reply:
Thanks Pawel, doing it now

Reply:
I noticed that there are other methods that do not adhere to this as well. 
I change to only those that I can - i.e adjacent methods while leaving the rest for some other time.